roadmap/implementers-guide/src/node/availability/bitfield-distribution.md

```diff
@@ -6,16 +6,24 @@ Validators vote on the availability of a backed candidate by issuing signed bitf
 
 `ProtocolId`: `b"bitd"`
 
-Input: [`BitfieldDistributionMessage`](../../types/overseer-protocol.md#bitfield-distribution-message)
+Input:
+[`BitfieldDistributionMessage`](../../types/overseer-protocol.md#bitfield-distribution-message) which are gossiped to all peers, no matter if validator or not.
+
 Output:
 
-- `NetworkBridge::RegisterEventProducer(ProtocolId)`
+- `NetworkBridge::RegisterEventProducer(ProtocolId)` in order to register ourselfs as an event provide for the protocol.
 - `NetworkBridge::SendMessage([PeerId], ProtocolId, Bytes)`
-- `NetworkBridge::ReportPeer(PeerId, cost_or_benefit)`
-- `BlockAuthorshipProvisioning::Bitfield(relay_parent, SignedAvailabilityBitfield)`
+- `NetworkBridge::ReportPeer(PeerId, cost_or_benefit)` improve or penalize the reputiation of peer based on the message we received relative to our state.
+- `BitfieldDistributionMessage::DistributeBitfield(relay_parent, SignedAvailabilityBitfield)` gossip a verified incoming bitfield on to interested subsystems within this validator node.
 
 ## Functionality
 
-This is implemented as a gossip system. Register a [network bridge](../utility/network-bridge.md) event producer on startup and track peer connection, view change, and disconnection events. Only accept bitfields relevant to our current view and only distribute bitfields to other peers when relevant to their most recent view. Check bitfield signatures in this subsystem and accept and distribute only one bitfield per validator.
+This is implemented as a gossip system. Register a [network bridge](../utility/network-bridge.md) event producer on startup.
+
+It is necessary to track peer connection, view change, and disconnection events, in order to maintain an index of which peers are interested in which relay parent bitfields.
```

Comment:
This section will be rendered as a single paragraph; it feels like it might be intended to be rendered as an unordered list.